lib/index.js

```diff
@@ -355,7 +355,7 @@ Server.prototype.of = function(name, fn){
  * @api public
  */
 
-Server.prototype.close = function(){
+Server.prototype.close = function(fn){
```

Comment:
I saw you updated the docs (thanks!), could you also update the jsdoc (above `@api public`, line 354-355) please?